addons/web/static/test/x2many.js

```diff
@@ -1,610 +0,0 @@
-odoo.define('web.test.x2many', function (require) {
```

Comment:
Ok, I did not look all pretty committed to make, that we do when the code is good